README.md

```diff
@@ -12,7 +12,6 @@ tldr; Get Started:
 1. Navigate to [Servless Application Repository](https://console.aws.amazon.com/serverlessrepo/home#/available-applications) and search for "athena-federation". Be sure to check the box to show entries that require custom IAM roles. 
 1. Look for entries published by the "Amazon Athena Federation" author.
 1. Deploy the application
-1. Go to the [Athena Console in us-east-1 (N. Virginia)](https://console.aws.amazon.com/athena/home) and create a workgroup called "AmazonAthenaPreviewFunctionality", any queries run from that workgroup will be able to use Preview features described in this repository. If you are running a federated query in US-East-1, Us-East-2, or Us-West-2, you can use any workgroup as long as you have set your engine setting to V2. See Engine Versions here:https://docs.aws.amazon.com/athena/latest/ug/engine-versions.html. 
```

Comment:
Can we revise this please? We should document that FQ is only available on V2.

Reply:
I removed the entire line since we still have 

```Athena Federated Queries are now enabled as GA in us-east-1, us-east-2, us-west-2, eu-west-1, ap-northeast-1, ap-south-1, us-west-1, ap-southeast-1, ap-southeast-2, eu-west-2, ap-northeast-2, eu-west-3, ca-central-1, sa-east-1, and eu-central-1. To use this feature, upgrade your engine version to Athena V2 in your workgroup settings. Check documentation here for more details: https://docs.aws.amazon.com/athena/latest/ug/engine-versions.html.```

earlier in the readme

Reply:
I think its fine here to reiterate, since its a walkthrough and we shouldn't have to expect users to jump back and forth tbh.